python/src/equistore/operations/equal.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+
+from equistore import TensorBlock, TensorMap
+
+from . import _dispatch
+from ._utils import _check_blocks, _check_maps, _check_same_gradients
+
+
+def equal(tensor1: TensorMap, tensor2: TensorMap, only_metadata: bool = False) -> bool:
+    """Compare two :py:class:`TensorMap`.
+
+    This function returns ``True`` if the two tensors have the same keys
+    (potentially in different order) and all the :py:class:`TensorBlock` have
+    the same (and in the same order) samples, components, properties,
+    and their values matrices pass the == test with the provided
+    ``rtol``, and ``atol``.
```

Comment:
There is no `atol`/`rtol` in this function parameters

Reply:
it is a copy-paste error

Reply:
i will correct it thanks